fastlane/Fastfile

```diff
@@ -38,20 +39,42 @@ EXTENSIONS_BUNDLE_IDENTIFIERS = %w[
   WordPressThisWeekWidget
   WordPressIntents
 ].map { |suffix| "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.#{suffix}" }
-
-ALL_BUNDLE_IDENTIFIERS = [APP_STORE_VERSION_BUNDLE_IDENTIFIER, *EXTENSIONS_BUNDLE_IDENTIFIERS]
-
-require_relative 'lanes/localization-lanes'
-
-# Use this instead of getting values from ENV directly
-# It will throw an error if the requested value is missing
+ALL_BUNDLE_IDENTIFIERS = [APP_STORE_VERSION_BUNDLE_IDENTIFIER, *EXTENSIONS_BUNDLE_IDENTIFIERS].freeze
+JETPACK_APP_IDENTIFIER = 'com.automattic.jetpack'
+
+# Environment Variables — used by lanes but also potentially actions
+Dotenv.load(USER_ENV_FILE_PATH)
+Dotenv.load(PROJECT_ENV_FILE_PATH)
+GHHELPER_REPO = 'wordpress-mobile/wordpress-iOS'
+ENV['PROJECT_NAME'] = 'WordPress'
+ENV['PUBLIC_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.Public.xcconfig')
+ENV['INTERNAL_CONFIG_FILE'] = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+ENV['DOWNLOAD_METADATA'] = './fastlane/download_metadata.swift'
+ENV['PROJECT_ROOT_FOLDER'] = "#{PROJECT_ROOT_FOLDER}/"
+ENV['APP_STORE_STRINGS_FILE_NAME'] = 'AppStoreStrings.po'
+ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'
+
+# Use this instead of getting values from ENV directly. It will throw an error if the requested value is missing
 def get_required_env(key)
-  unless ENV.key?(key)
-    UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?")
-  end
+  UI.user_error!("Environment variable '#{key}' is not set. Have you setup #{USER_ENV_FILE_PATH} correctly?") unless ENV.key?(key)
   ENV[key]
 end
```

Comment:
This method is also used in the lanes we import below. I wonder if it would be more appropriate to define it in a `shared.rb` file that the lanes themselves import but that this one doesn't know about?

Although... maybe it's too much effort. At the end of the day, the lanes ~~will~~ should always and only be called via this `Fastfile`, so the result is the same.